### PR TITLE
Docs copyedits: README.md + CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by khorne3**
The first in a series of edits to the code-server docs.

Changes include copy edits and basic structural reorg to README.md, CONTRIBUTING.md for clarity. All feedback regarding changes welcome.

~If it's not a problem, I'd like to move contributing to the root level a la other open-source projects.~

### CODE REVIEW

**Comment by nhooyr** [comment]
> If it's not a problem, I'd like to move contributing to the root level a la other open-source projects.

Sounds good!

**Review by nhooyr — COMMENTED** [review]
Could you justify some of these changes? It's not clear to me what was improved.

File: doc/CONTRIBUTING.md
Comment (nhooyr) [inline doc/CONTRIBUTING.md:5]: Use `yarn fmt` to format this, it'll prevent the extra nesting.

**Comment by khorne3** [comment]
@nhooyr Sure -- was there anything in particular that stood out for you? My goal is to make docs as clear as possible. The copy edits (spelling, grammar, etc.) are self-explanatory. I had to read some things multiple times to understand, and in some parts, it seemed the context/explanations were reversed.

File: README.md
Comment (nhooyr) [inline README.md:18]: Link here looks wrong.

File: README.md
Comment (nhooyr) [inline README.md:50]:
```suggestion
See [CONTRIBUTING](./doc/CONTRIBUTING.md) for details.
```

File: doc/CONTRIBUTING.md
Comment (nhooyr) [inline doc/CONTRIBUTING.md:11]: I like this!

**Comment by nhooyr** [comment]
> @nhooyr Sure -- was there anything in particular that stood out for you? My goal is to make docs as clear as possible. The copy edits (spelling, grammar, etc.) are self-explanatory. I had to read some things multiple times to understand (I try to look at things from the perspective of someone who is completely new), and in some parts, it seemed the context/explanations were reversed.

Fair enough! As writer of these docs, I'm very biased in my opinion and understanding of them so I'll defer to you. I'll add some comments.

File: doc/CONTRIBUTING.md
Comment (nhooyr) [inline doc/CONTRIBUTING.md:129]:
```suggestion
the VS Code front-end and implements the server. You can find this in
```

File: doc/CONTRIBUTING.md
Comment (nhooyr) [inline doc/CONTRIBUTING.md:147]: This line needs to be wrapped shorter.

File: doc/CONTRIBUTING.md
Comment (nhooyr) [inline doc/CONTRIBUTING.md:134]: Appreciate the grammar fixes here.

**Review by nhooyr — CHANGES_REQUESTED** [review]
Overall 👍🏼

**Review by nhooyr — APPROVED** [review]
Thanks @khorne3

**Comment by khorne3** [comment]
@nhooyr What do I need to do re: failing checks?

File: doc/CONTRIBUTING.md
Comment (nhooyr) [inline doc/CONTRIBUTING.md:1]: People may be using this path actively in links and whatnot and us moving it will break.

I think we should just keep it as is.

**Comment by nhooyr** [comment]
@khorne3 you can ignore those, I'll merge 4 you after my last comment is addressed

**Comment by nhooyr** [comment]
Thanks again @khorne3

**Pull request merged**
